Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -167,6 +165,41 @@
       reagents:
       - ReagentId: JuiceBanana
         Quantity: 10
+  - type: SpawnItemsOnUse
+    items:
+    - id: FoodBananaPeeled
+    sound:
+      path: /Audio/Effects/unwrap.ogg
+
+- type: entity
```

Comment:
Why does this copy/paste so much of `FoodBanana` instead of parenting from it?

Reply:
Because otherwise it would inherit the dpaenitemonuse component and create an infinite chain of peeling an already peeled banana

Reply:
I see, good catch. What if you set the number of uses in the peel to zero?

(Setting number of uses to zero is currently broken in C#, but I'm going to fix that now.)

Reply:
(Even without the fix, you can make the original banana parent from the original peeled banana.)